.github/workflows/upload_pr_documentation.yml

```diff
@@ -0,0 +1,16 @@
+name: Upload PR Documentation
```

Comment:
Not sure why but looks like this workflow is not triggered once "Build PR Documentation" completed

Reply:
should be fixed with https://github.com/huggingface/optimum-executorch/pull/22 thanks @regisss :goat: